src/app/utilities/Toaster.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { expect } from 'chai';
+import { ToastyService, ToastOptions, ToastData, ToastyConfig } from 'ng2-toasty';
+import { Toaster } from './Toaster';
+
+describe('Toaster', () => {
+    let mockedToastyService: ToastyService;
+    let sut: Toaster;
+    let passedOptions: ToastOptions | string | number;
+
+    beforeEach(() => {
+        mockedToastyService = new ToastyService(new ToastyConfig());
+        sinon.stub(mockedToastyService, 'default')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
+        sinon.stub(mockedToastyService, 'info')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
+        sinon.stub(mockedToastyService, 'success')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
+        sinon.stub(mockedToastyService, 'wait')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
+        sinon.stub(mockedToastyService, 'warning')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
+        sinon.stub(mockedToastyService, 'error')
+            .callsFake((options: ToastOptions | string | number) => { passedOptions = options; });
```

Comment:
oh wow. You need to talk me through this syntax. I've not figured it out yet. is that an or?